src/index.js

```diff
@@ -126,6 +132,53 @@ class File {
   }
 }
 
+class FileInterface extends File {
+
+  get displayName() {
+    return "FileInterface";
+  }
+
+  // create and return a path url
+  get path() {
+    return  URL.createObjectURL(this.descriptor);
+  }
+
+  get encoding() {
+    return DEFAULT_ENCODING;
+  }
+
+  get size() {
+    return this.descriptor.size;
+  }
+
+  get fileName() {
+    return this.descriptor.name;
+  }
+
+  // call content function and generate MD5 hash from this content
+  async hash() {
+    const content = await this.contentArrayBuffer()
+    const wordArray = CryptoJS.lib.WordArray.create(content);
```

Comment:
Is there any specific reason using CryptoJS? I mean in another place `FileLocal` we use `crypto` in `hash()` function, maybe we can reuse the same code and module?

Reply:
Is the same situation as the dependencies above, I tried to use the same logic using just crypto but didn't work for the file come from the input(browser).
But if you have any idea, we can try.